### DISCUSSION TITLE
[BUG] Cursor jumps when trying to paste at end of text in description field in edit view

### DISCUSSION THREAD

**Discussion by graue70** [TechbeeAT/jtxBoard#709] [General]
**Describe the bug**
The cursor jumps and selects the first word when trying to paste at the end of existing text in the description field in edit view.

**To Reproduce**
1. Copy some text from anywhere (other app, doesn't matter)
2. Open/create a task and go to the edit view
3. Input some text into the description
4. Press long at the end of the text. (This is supposed to reveal the 'paste' button.)
5. The first word of the description field is suddenly selected. (Note that it's not always, but usually, reproducible. I haven't found a pattern yet.)

Also, if I try to paste in an unordered list, the cursor jumps to a seemingly random place in the list and selects a seemingly random amount of text.

**Expected behavior**
The cursor should be placed where I touched.

**Device and version:**
 - Device: Samsung Galaxy S10
 - Android version: Android 13
 - jtx Board version: 2.4.7.ose
 - Downloaded from: f-Droid

**Comment by patrickunterwegs** [comment]
Hi @graue70 , I can't reproduce this issue. Do you use any specific keyboard? Could you make try it also with another one?

**Comment by graue70** [comment]
I use 'Anroid-Tastatur (AOSP)' (com.android.inputmethod.latin) which is the default that came with my Lineage OS.

**Comment by patrickunterwegs** [comment]
If you turn off markdown formatting through the menu on the top, does it have the same behavior? Do you have the same behavior in other fields than the description field?

**Comment by graue70** [comment]
Yes and yes. Also happens in the title field.

**Comment by rogercreagh** [comment]
I have two similar problem using OpenBoard keyboard. Not when pasting, but these:
1.
Type some text.
Tap to move the cursor back to make a correction
Correct the text and tap at the end of the text body to continue adding text
Start typing - the cursor jumps back to the point where the text was corrcted, deletes the corrected text and starts to add the text there.
2.
Type some text
Use backspace delete to delete the last word
Sometimes after 2 or more backspace-deletes it deletes the space after the previous word, concatenating the previous word and the remains of the word being edited.

I did not think that either of these were jtxB related - more a (annoying) quirk of the keyboard and/or slow response by the device if something else was happening in the background (eg keyboard buffer not being flushed if DavX5 happened to be doing a sync, or some other app doing something.

**Comment by patrickunterwegs** [comment]
It's the same situation as in #684 , I can't reproduce it and there is no logic manipulating texts in other fields than the description field. As you say it also happens on the summary field, where there is really nothing happening in the background, I can only assume that this is an issue coming from the keyboard...

**Comment by patrickunterwegs** [comment]
I will move this topic to discussions as I don't see a way to get this fixed within jtx Board.
